lib/bitex_bot/models/opening_flow.rb

```diff
@@ -34,16 +34,15 @@ def self.create_for_market(remote_balance, order_book, transactions, bitex_fee,
       self.store = store
 
       remote_value, safest_price = calc_remote_value(bitex_fee, other_fee, order_book, transactions)
-      raise CannotCreateFlow, "Needed #{remote_value} but you only have #{remote_balance}" if remote_value > remote_balance
+      raise CannotCreateFlow, "Needed #{remote_value} but you only have #{remote_balance}" unless
+         enough_remote_funds?(remote_balance, remote_value)
 
-      bitex_price = bitex_price(value_to_use, remote_value)
+      bitex_price = bitex_price(remote_value) * fx_rate
       order = create_order!(bitex_price)
       raise CannotCreateFlow, "You need to have #{value_to_use} on bitex to place this #{order_class.name}." unless
         enough_funds?(order)
 
-      Robot
-        .logger
-        .info("Opening: Placed #{order_class.name} ##{order.id} #{value_to_use} @ $#{bitex_price} (#{remote_value})")
+      Robot.log(:info, "Opening: Placed #{order_class.name} ##{order.id} #{value_to_use} @ $#{bitex_price} (#{remote_value})")
```

Comment:
This order was placed on Bitex, not necessarily BTC/USD.

Reply:
was done